src/k8s/cmd/k8sd/k8sd.go

```diff
@@ -33,8 +33,9 @@ func addCommands(root *cobra.Command, group *cobra.Group, commands ...*cobra.Com
 
 func NewRootCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "k8sd",
-		Short: "Canonical Kubernetes orchestrator and clustering daemon",
+		Use:    "k8sd",
+		Short:  "Canonical Kubernetes orchestrator and clustering daemon",
+		PreRun: cmdutil.HookCheckLXD(true),
```

Comment:
The user does not invoke this (this is done by the snap). No need to add this here.